packages/blockchain-wallet-v4-frontend/src/store/index.js

```diff
@@ -46,12 +48,9 @@ const devToolsConfig = {
   serialize: serializer
 }
 
-const configureStore = async function () {
+const configuredStore = async function () {
   const history = createHashHistory()
   const sagaMiddleware = createSagaMiddleware()
-  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
-    ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__(devToolsConfig)
-    : compose
```

Comment:
Does devtools still work?

Reply:
yeah part of redux-toolkit is that devtools is add/configured automatically.